### ISSUE TITLE
Validation Splits

### ISSUE THREAD

**Issue by m-parchami** [megvii-research/mdistiller#45]
Hi,

Thanks for the great repo!

I was wondering if for CIFAR-100 and Imagenet there's also a "validation" (as a heldout subset of training set) is being created somewhere in the code? Because as I looked for it, it seems that we only have a `train_dataset` and `test_dataset`.

All the best.
Thanks again for the code :)

**Comment by Zzzzz1** [comment]
The ```test_dataset``` in the repo is actually the validation set that we use 😂.
